yaq_traits/has-transformed-position.toml

```diff
@@ -0,0 +1,73 @@
+trait = "has-transformed-position"
+doc = """
```

Comment:
Broadly, filling out of docstrings in this file would be appreciated

Also should include the `requires` field